Code/Framework/AzFramework/Platform/Linux/AzFramework/Process/ProcessCommunicator_Linux.cpp

```diff
@@ -38,12 +68,40 @@ namespace AzFramework
     void CommunicatorHandleImpl::Close()
     {
         close(m_handle);
+        if (m_epollHandle != -1)
```

Comment:
Should we check if `m_handle` is closed as well and do the similar logic?

Reply:
We weren't before, and I've tried to touch as little as possible.  its not checking the result value of close, and you can't close handle -1.  So worst case this would just result in errno being set.